make/lib/Awt2dLibraries.gmk

```diff
@@ -869,7 +870,8 @@ ifeq ($(ENABLE_HEADLESS_ONLY), false)
       EXTRA_HEADER_DIRS := $(LIBSPLASHSCREEN_HEADER_DIRS), \
       DISABLED_WARNINGS_gcc := sign-compare type-limits unused-result \
           maybe-uninitialized shift-negative-value implicit-fallthrough, \
-      DISABLED_WARNINGS_clang := incompatible-pointer-types deprecated-declarations, \
+      DISABLED_WARNINGS_clang := incompatible-pointer-types deprecated-declarations \
```

Comment:
/clean I manually fixed conflict in this line since the jdk11 has a different list of disabled warnings, also the dates in the copyrights are different.